2-mandatory/3-shopping-list.js

```diff
@@ -19,7 +19,16 @@ let pantry = {
 };
 
 function createShoppingList(recipe) {
-  // write code here
+  const shoppingList = { name: recipe.name, items: [] };
+  for (const ingredient of recipe.ingredients) {
```

Comment:
have you run the tests? `ingredient` will change at each iteration so using `const` will give you an error